appautoscaling.tf

```diff
@@ -26,3 +26,23 @@ resource "aws_appautoscaling_policy" "scale_cpu" {
     }
   }
 }
+
+resource "aws_appautoscaling_policy" "scale_mem" {
+  count              = var.launch_type == "FARGATE" ? 0 : var.autoscaling_memory ? 1 : 0
+  name               = "scale-mem"
```

Comment:
This name can be changed to `scale-memory`.

Reply:
Adjusted